### PR TITLE
using maps parameter for reverse intensity

### PR DESCRIPTION
**Pull request by waxenegger**
Following suit with https://github.com/openmicroscopy/openmicroscopy/pull/5082.

TEST: https://cowfish.openmicroscopy.org/webtrial
Check that the &maps= parameter is part of the image requests and that setting reverse is applied to the channels in question. Also check that the copy and paste, as well as the save functionality is working.

### CODE REVIEW

**Comment by waxenegger** [comment]
--exclude

**Comment by will-moore** [comment]
URLs look good. Copy, paste & save working fine. Good to merge.

File: src/utils/constants.js [hunk 1]
```diff
@@ -40,61 +40,62 @@ export const WEBCLIENT = "WEBCLIENT";
...
-    IMAGE_ID : 'IMAGE_ID',
-    CHANNELS : 'C',
-    PLANE : 'Z',
-    TIME : 'T',
-    PROJECTION : 'P',
-    MODEL : 'M',
-    CENTER_X : 'X',
-    CENTER_Y : 'Y',
-    ZOOM : 'ZM'
+    CHANNELS: 'C',
```
Comment (jburel) [inline src/utils/constants.js]: Not possible to use the constants defined in ol3-viewer/src/ome/ol3/globals.js
only diff is ``DATASET_ID``?

Comment (waxenegger) [inline src/utils/constants.js]: @jburel one is the internal ol3-viewer which is compiled by closure and can be run by itself. so they also have different namespacing, ecma script standards and even code. I'm not super happy about the duplication in some places but it's the nature of the beast.

Comment (jburel) [inline src/utils/constants.js]: Thanks for the clarification

**Pull request merged**